test/StreamTest.php

```diff
@@ -55,6 +58,14 @@ protected function tearDown(): void
         if (is_string($this->tmpnam) && file_exists($this->tmpnam)) {
             unlink($this->tmpnam);
         }
+
+        foreach (self::$tempFiles as $tempFile) {
+            if (! file_exists($tempFile)) {
+                continue;
+            }
+
+            unlink($tempFile);
+        }
```

Comment:
This is wrong. those temp files are created by a provider which is executed before all tests and such this must be cleaned after all tests.